### PR TITLE
Checkout form HTML/CSS

### PR DESCRIPTION
**Pull request by yulyasystem**
### HTML5 Form

[Demo](https://lucid-torvalds-61553a.netlify.com/) |
[Code base](https://github.com/yulyasystem/checkout-page)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/yulyasystem/checkout-form/index.html [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+          </tr>
+          <tr>
+            <td class="order-td">2 documents</td>
+            <td class="order-td">2 languages</td>
+            <td class="order-td">$ 696.06</td>
+            <td class="order-td">48 hours*</td>
+          </tr>
+        </table>
+        <div class="header-buttons">
+          <label for="submit-button" class=" button order-button" tabindex="0">
```
[the remark this answers is not in the document]
Comment (yulyasystem) [reply submissions/yulyasystem/checkout-form/index.html]: Thank you! I made inputs and radio-buttons accessible from keyboard.

File: submissions/yulyasystem/checkout-form/index.html [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+
+
+      </div>
+
+      <div class="card-detail-section">
+
+        <div class="card-field card-number">
+          <label for="number" class="card-label">Card number</label>
+          <input type="text"  name="card number" id="number" 
+            minlength="16" maxlength="16" class="card-input input-name " required>
```
[the remark this answers is not in the document]
Comment (yulyasystem) [reply submissions/yulyasystem/checkout-form/index.html]: Ooops! :scream_cat: Fix all patterns for inputs :)

File: submissions/yulyasystem/checkout-form/index.html [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+        </div>
+      </fieldset>
+
+      <div class="upload-document-section">
+        <div class="upload-heading-section">
+          <span class="circle"> 1</span>
+          <b class="upload-heading">Upload your documents</b>
+        </div>
+        <input type="file" name="upload" id="input-file" class=" upload-input visually-hidden" />
+        <label for="input-file" class="upload-label-file" >
```
Comment (yevhenorlov) [inline submissions/yulyasystem/checkout-form/index.html]: did you update the demo? I still can see the tabindex here 🤔 
image

File: submissions/yulyasystem/checkout-form/index.html [hunk 4]
```diff
@@ -44,12 +44,12 @@
           </tr>
         </table>
         <div class="header-buttons">
+          <input type="submit" value="" id="submit-button" class="submit-input visually-hidden">
           <label for="submit-button" class=" button order-button" tabindex="0">
```
[the remark this answers is not in the document]
Comment (yulyasystem) [reply submissions/yulyasystem/checkout-form/index.html]: fixed it :)

**Pull request merged**